Makefile

```diff
@@ -24,6 +24,8 @@
 
 VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
 MOD_ID = $(shell cat user.config mod.config 2> /dev/null | awk -F= '/MOD_ID/ { print $$2; exit }')
+MOD_VERSION = $(shell cat user.config $(MANIFEST_PATH) 2> /dev/null | awk -F':' '/Version/ { gsub(/[[:blank:]]/, "", $$2); print $$2; exit }')
```

Comment:
We can also use run awk directly on the manifest, cat function is just copied from above. Separator `:`, print second field for match `Version` and trim leading/trailing spaces.

Reply:
The idea in the other lines is to con`cat`enate the mod and user configs so that we can parse them both in one statement (with `user.config` taking priority).  Combining `user.config` and `mod.yaml` in this definition looks bogus, did you really intend that?

Reply:
Changed to call the manifest file directly.